### PR TITLE
Update apis

### PR DESCRIPTION
**Pull request by nt-gt**

### CODE REVIEW

File: ebl/v3/issuance/EBL_ISS_v3.0.0-Beta-3.yaml [hunk 1]
```diff
@@ -227,9 +227,21 @@ components:
           $ref: '#/components/schemas/IssueToParty'
         eBLVisualisationByCarrier:
           $ref: '#/components/schemas/SupportingDocument'
+        issuanceManifestSignedContent:
+          type: string
+          pattern: '^([a-zA-Z0-9_=]+)\.([a-zA-Z0-9_=]+)\.([a-zA-Z0-9_\-\+\/=]+)$'
```
Comment (HenrikHL) [inline ebl/v3/issuance/EBL_ISS_v3.0.0-Beta-3.yaml:232]:
```suggestion
          pattern: ^([a-zA-Z0-9_=]+)\.([a-zA-Z0-9_=]+)\.([a-zA-Z0-9_\-\+\/=]+)$
```

File: pint/v3/EBL_PINT_v3.0.0-Beta-3.yaml [hunk 2]
```diff
@@ -540,6 +571,15 @@ components:
             actual type: [`DocumentMetadata`](#\DocumentMetadata)
 
             This attribute is used to transfer metadata of the **'Digital copy of the original physical B/L document' document** a.k.a. **eBLVisualisationByCarrier document**. If this attribute is defined with the initial eBL envelope transfer(in the first `EnvelopeTransferChainEntrySignedContent` entry in the [`EblEnvelope.envelopeTransferChain[]`](#/EblEnvelope) list), eBLVisualisationByCarrier document **must** be transferred with every subsequent envelope transfer for the lifetime of the eBL document. Also, eBLVisualisationByCarrier document must be unchanged between different envelope transfers for the lifetime of the eBL document (in other words, for each `EnvelopeTransferChainEntrySignedContent` entry in the [`EblEnvelope.envelopeTransferChain[]`](#/EblEnvelope) list, JWS-signed payload `EnvelopeTransferChainEntry.eBLVisualisationByCarrier` attribute must have the identical value).
+        IssuanceManifestSignedContent:
+          type: string
+          pattern: '^([a-zA-Z0-9_=]+)\.([a-zA-Z0-9_=]+)\.([a-zA-Z0-9_\-\+\/=]+)$'
```
Comment (HenrikHL) [inline pint/v3/EBL_PINT_v3.0.0-Beta-3.yaml:576]:
```suggestion
          pattern: ^([a-zA-Z0-9_=]+)\.([a-zA-Z0-9_=]+)\.([a-zA-Z0-9_\-\+\/=]+)$
```

File: ebl/v3/issuance/EBL_ISS_v3.0.0-Beta-3.yaml [hunk 3]
```diff
@@ -254,7 +254,7 @@ components:
           type: string
           format: byte
           description: The actual contents of the visual rendering.
-      required:
+          example: iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsQAAA7EAZUrDhsAAAGaSURBVDhPnZO/S0JRFMe/zygxwgqcfZtz0N7SFNgPaKlJpTFLCqwotV9qRYN/gIOL1FK22NISWENT0BTUkNLgIL2iHxYRnc697/kKzdA+cOD8uOec77uXB/oHngMPnd2eSb/pAf5DP2EWhGlQ8ChIikiiQa7vruFacwHdHHwC9nY7mhqgRBTdsbDdA/nVvHQbYnxvHHhnp4XtFZjon4DapTam4Lx4jt7NXlO6WEsreltDA5RFlt4qHDaWXlgrwNnplDX5CcWnIo5vjmWimsGdQV7HjjjJ0gMDAbNZopU1wgwfmQSlL9JCkEkunyMEuLbMFgZZ161G5RsFES5WNrC8lC8Fb49XDlcWWLNVOHqttFGCo90haxUsyeEk8GhEfEm+lA/ZqyyGdof0ocJegMhIpKZZIC8xfhLH0v6SfstCzRubeK42tg9Od3RDm9c4qMV8hWguinAmDHTJvC5bVB6A8nYZtlabTFcjX0EQ6gshNhqTDSbPQGIsUbdZIhT8ZOt0izDFu+dAakI1svX59W/MXGbIveM2or8g+gL+Fn3DwcYf+gAAAABJRU5ErkJggg==      required:
```
Comment (nt-gt) [inline ebl/v3/issuance/EBL_ISS_v3.0.0-Beta-3.yaml:257]: FYI, this change lead to broken YAML. :) (Note the `required:` being part of the example)

Comment (nt-gt) [reply ebl/v3/issuance/EBL_ISS_v3.0.0-Beta-3.yaml:257]: (Will fix with my merge)

Comment (HenrikHL) [reply ebl/v3/issuance/EBL_ISS_v3.0.0-Beta-3.yaml]: Sorry...

**Review by HenrikHL — CHANGES_REQUESTED** [review]
I don't think we should update Beta 3 any more. The idea going forward is that we update the main version and then once in a while make a snapshot.
All updates should be done to the main versions (v3.0.0 and not the Beta 3). For the RSP (Response) APIs - these APIs should be deprecated

**Comment by HenrikHL** [comment]
Closing this PR as it is part of #301

**Pull request closed without merging**